.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: File a bug report for hypothesis' browser-extension.
+title: 'Bug report'
+labels: 'triage_bug'
```

Comment:
I suggest it would make sense to separate the _type_ of issue from whether it needs triaging by a Hypothesis team member or not. So for example I would suggest to use two labels here: `Needs triage` + `bug` (already exists). Also a note on label naming - please make sure the style is consistent with one of the two styles that existing labels use. See https://github.com/hypothesis/browser-extension/labels. They are either "Sentence case" or "all lower case".